filament/backend/src/opengl/OpenGLContext.h

```diff
@@ -50,10 +50,12 @@ class OpenGLContext {
         // VertexBuffer supports buffer objects. If this is zero, then the VBO handles array is
         // immutable.
         backend::Handle<backend::HwVertexBuffer> vertexBufferWithObjects = {};
+        backend::Handle<backend::HwIndexBuffer> indexBuffer = {};
 
         // If this version number does not match vertexBufferWithObjects->bufferObjectsVersion,
         // then the VAO needs to be updated.
         uint8_t vertexBufferVersion = 0;
+        uint8_t indexBufferVersion = 0;
```

Comment:
Comment might need an update: what is `indexBufferVersion` compared against (and why)?

Reply:
fixed in https://github.com/shapr3d/filament/pull/89/commits/19d96e5114fbf8884f3556634a95fd3018a938ab